README.md

```diff
@@ -45,7 +48,7 @@ var add = (x, y) => {
 }
 ```
 
--- YOUR ANSWER HERE --
+ES6, add is not anonymous, add
```

Comment:
Fat arrow functions as always anonymous because the have no name. Call it as `add(2.4,3.3333)`